services/wishlistService.js

```diff
@@ -0,0 +1,24 @@
+const wishlistDao = require('../models/wishlistDao');
+
+const getWishlist = async(userId) => {
+  return await wishlistDao.getWishlist(userId);
+};
+
+const deleteWishlist = async(userId, productId) => {
+  return await wishlistDao.deleteWishlist(userId, productId);
+};
+
+const addWishlist = async(userId, productId) => {
+  const checkWishlist = await wishlistDao.checkWishlist(userId, productId)
+  if(!checkWishlist.length){
```

Comment:
checkWishList에 대한 정보 유무에 따른 분기처리는 다음과 같이 작성하여 더욱 간결하게 작성할 수 있을 것 같습니다!
```suggestion
  if(!checkWishlist){
```